docs/source/conf.py

```diff
@@ -193,7 +193,10 @@
 # -- Options for intersphinx extension ---------------------------------------
 
 # Example configuration for intersphinx: refer to the Python standard library.
-intersphinx_mapping = {"https://docs.python.org/3/": None}
+intersphinx_mapping = {
+    "python": ("https://docs.python.org/3", None),
```

Comment:
I think I've already seen such change in another PR :)

Reply:
Yes, this is needed to render properly. I made the change exactly the same, so there won't be conflicts.